examples/MQ/pixelDetector/src/devices/FairMQPixelFileSink.cxx

```diff
@@ -68,7 +68,7 @@ bool FairMQPixelFileSink::StoreData(fair::mq::Parts& parts, int /*index*/)
 
     cleanup.reserve(numParts);
     objectsForBranches.resize(numParts, nullptr);
-    for (int ipart = 0; ipart < numParts; ipart++) {
+    for (decltype(parts.Size()) ipart = 0; ipart < numParts; ipart++) {
```

Comment:
Wouldn't this better fit in the sign-compare commit?

It's not quite related to fChannels at all?

Reply:
Ups, indeed, probably chose the wrong commit in a rebase fixup, thx

Reply:
done